compiler/src/Language/Mimsa/Actions/Optimise.hs

```diff
@@ -0,0 +1,20 @@
+module Language.Mimsa.Actions.Optimise where
+
+import qualified Data.Set as S
+import qualified Language.Mimsa.Actions.Monad as Actions
+import Language.Mimsa.Transform.FindUnused
+import Language.Mimsa.Transform.TrimDeps
+import Language.Mimsa.Types.AST
+import Language.Mimsa.Types.Store
+
+-- | given an expression, optimise it and create a new StoreExpression
+-- | this now accepts StoreExpression instead of expression
+optimise ::
```

Comment:
Will need a version of this that finds by name and updates it in place for use in repl / UI.